### PR TITLE
[Flutter] Non EVM support docs

### PR DESCRIPTION
**Pull request by quetool**
Update docs to reflect non-EVM support

### CODE REVIEW

File: docs/appkit/flutter/core/options.mdx [hunk 1]
```diff
@@ -35,20 +37,26 @@ Enable analytics to get more insights on your users activity within your [Reown
...
 ### featuresConfig:
 
-Used to configure extra features such as Analytics, [Email and Social Login](../core/email.mdx).
+Used to configure extra features such [Email and Social Login](../core/email.mdx).
 
-### requiredNamespaces: and optionalNamespaces:
+### getBalanceFallback:
 
-These values are optionals and, in most cases, not required since AppKit already defines every required and optional namespace internally based on configured networks. 
+This callback method will be triggered if getting the balance from our blockchain API fails due to unsupported network. You should place here your own getBalance() method if you want.
```
Comment (rtomas) [inline docs/appkit/flutter/core/options.mdx:50]:
```suggestion
This callback method will be triggered if getting the balance from our blockchain API fails due to unsupported network. You may replace this with your own `getBalance()` method if desired.
```

File: docs/appkit/flutter/core/siwe.mdx [hunk 2]
```diff
@@ -19,9 +19,14 @@ Connecting a wallet, proving control of an address with an off-chain signature,
 WalletConnect uses permissions expressed as ReCaps to enable a One-Click Authentication.
 
 ## Pre-requisites
-In order for SIWE to work, you need to have a backend to communicate with. This backend will be used to generate a nonce, verify messages and handle sessions.
+
+In order for 1-CA + SIWE to work, you need to have a backend to communicate with. This backend will be used to generate a nonce, verify messages and handle sessions.
```
Comment (rtomas) [inline docs/appkit/flutter/core/siwe.mdx:23]:
```suggestion
For 1-CA + SIWE to function properly, a backend for communication is required. This backend will be used to generate a nonce, verify messages and handle sessions.
```

File: docs/appkit/flutter/core/siwe.mdx [hunk 3]
```diff
@@ -19,9 +19,14 @@ Connecting a wallet, proving control of an address with an off-chain signature,
...
 More info [here](https://docs.login.xyz/sign-in-with-ethereum/quickstart-guide/implement-the-backend)
 
+:::info note
+This feature is compatible only with EVM blockchains, so if you decide to included non-EVM blockchains 1-CA + SIWE mechanism is going to be disabled internally.
```
Comment (rtomas) [inline docs/appkit/flutter/core/siwe.mdx:27]:
```suggestion
This feature is compatible only with EVM blockchains. Therefore, including non-EVM blockchains will result in the internal disabling of the 1-CA + SIWE mechanism.
```

I prefer not to use the word you or try to find another way

File: docs/appkit/flutter/core/custom-chains.mdx [hunk 4]
```diff
@@ -2,46 +2,80 @@
 
 ## Custom Networks addition and selection
 
-AppKit already comes with a list of supported networks inside of `ReownAppKitModalNetworks` class. You can get default supported EVM networks by selecting `ReownAppKitModalNetworks.supported['eip155']` and, currently, only EVM Networks are supported (non-EVM support will come soon). Check the list of [supported networks](https://github.com/walletconnect/blockchain-api/blob/master/SUPPORTED_CHAINS.md#list-of-supported-chains)
+AppKit already comes with a list of supported mainnets (including Solana mainnet) and testnets (including Solana testnet) inside of `ReownAppKitModalNetworks` class. This means that if you are looking to support EVM chains + Solana then there's nothing more you should tweak.
```
[the remark this answers is not in the document]
Comment (quetool) [reply docs/appkit/flutter/core/custom-chains.mdx]: Done!

**Review by rtomas — APPROVED** [review]

**Pull request merged**